### ISSUE TITLE
release readiness

### ISSUE THREAD

**Issue by tohuwabohu-io** [tohuwabohu-io/librefit#72]
- set up container registry 
- add release workflow
- build and publish librefit-service and librefit-web docker images

**Comment by tohuwabohu-io** [comment]
I experimented with GraalVM and initially planned to use the native or native-micro image. Sadly that's not possible atm.

Some insights:
- flyway migration added: https://flywaydb.org/ the quarkus plugin provided some convenience
- flyway has limitations for reactive datasources, so both the JDBC and the reactive-pg driver need to be present. This causes some warnings on startup
- flyway needs some workarounds. See io.tohuwabohu.util.FlywayMigration.kt

Those workarounds cause troubles with the migration during runtime. I tried some tricks but in the end ended up in a situation where I'd have to manually maintain a migration SQL list in the application.properties file just to make a really wonky-looking workaround for other workarounds work.

Other stuff:
- updated various dependencies for librefit-web
- updated node version from 19.something -> 21.7.0
- added adapter-node to librefit-web and dockerized the SvelteKit app
- changed PUBLIC_API_BASE_PATH -> API_BASE_PATH and import from $env/dynamic/private. That way the service URL can be changed for each deployment

For testing the whole stack, a new docker-compose file has been created in deployment/test-delivery
